src/frontend/Komponenter/Behandling/Simulering/SimuleringTyper.ts

```diff
@@ -8,6 +8,7 @@ export interface ISimulering {
     forfallsdatoNestePeriode: string;
     tidSimuleringHentet: string;
     tomSisteUtbetaling: string;
+    sumManuellePosteringer?: number;
```

Comment:
```suggestion
    antallManuellePosteringer?: number;
```

Hadde antall passet bedre her?

Reply:
Vi summerer beløpet

Reply:
Kunne forsåvidt ha bare summert antallet

Reply:
ahh, jeg misforstod, sum blir vel riktig da, ja